DFC.ServiceTaxonomy.Editor/Recipes/245.JobProfileCollegeentryrequirements.recipe.json

```diff
@@ -0,0 +1,158 @@
+{
+  "name": "JobProfileCollegeentryrequirements",
```

Comment:
Is there an issue if we make it PascalCase like JobProfileCollegeEntryRequirements for every occurance of this word within this file?